website_sale_collapse_categories/static/src/css/website_sale.css

```diff
@@ -1,8 +1,38 @@
-#o_shop_collapse_category li a{
-  display: inline-block;
-  width: 80%;
-  padding-left: 3px;
-}
-#o_shop_collapse_category li i.fa{
-  cursor: pointer;
-}
+/*
+Error: Inconsistent indentation: 1 space was used for indentation, but the rest of the document was indented using 4 spaces.
+        on line 9 of /app/website/website_sale_collapse_categories/static/src/css/website_sale.sass
+
+4:             display: inline-block
+5:             width: 80%
+6:             padding-left: 3px
+7: 
+8:         i.fa
+9:  cursor: pointer
+
+Backtrace:
+/app/website/website_sale_collapse_categories/static/src/css/website_sale.sass:9
+/usr/local/bundle/gems/sass-3.4.16/lib/sass/engine.rb:483:in `block in tabulate'
+/usr/local/bundle/gems/sass-3.4.16/lib/sass/engine.rb:441:in `each'
+/usr/local/bundle/gems/sass-3.4.16/lib/sass/engine.rb:441:in `each_with_index'
+/usr/local/bundle/gems/sass-3.4.16/lib/sass/engine.rb:441:in `tabulate'
+/usr/local/bundle/gems/sass-3.4.16/lib/sass/engine.rb:406:in `_to_tree'
+/usr/local/bundle/gems/sass-3.4.16/lib/sass/engine.rb:378:in `_render_with_sourcemap'
+/usr/local/bundle/gems/sass-3.4.16/lib/sass/engine.rb:295:in `render_with_sourcemap'
+/usr/local/bundle/gems/sass-3.4.16/lib/sass/plugin/compiler.rb:490:in `update_stylesheet'
+/usr/local/bundle/gems/sass-3.4.16/lib/sass/plugin/compiler.rb:215:in `block in update_stylesheets'
+/usr/local/bundle/gems/sass-3.4.16/lib/sass/plugin/compiler.rb:209:in `each'
+/usr/local/bundle/gems/sass-3.4.16/lib/sass/plugin/compiler.rb:209:in `update_stylesheets'
+/usr/local/bundle/gems/sass-3.4.16/lib/sass/plugin/compiler.rb:293:in `watch'
+/usr/local/bundle/gems/sass-3.4.16/lib/sass/plugin.rb:108:in `method_missing'
+/usr/local/bundle/gems/sass-3.4.16/lib/sass/exec/sass_scss.rb:384:in `watch_or_update'
+/usr/local/bundle/gems/sass-3.4.16/lib/sass/exec/sass_scss.rb:51:in `process_result'
+/usr/local/bundle/gems/sass-3.4.16/lib/sass/exec/base.rb:52:in `parse'
+/usr/local/bundle/gems/sass-3.4.16/lib/sass/exec/base.rb:19:in `parse!'
+/usr/local/bundle/gems/sass-3.4.16/bin/sass:13:in `<top (required)>'
+/usr/local/bundle/bin/sass:23:in `load'
+/usr/local/bundle/bin/sass:23:in `<main>'
+*/
+body:before {
+  white-space: pre;
+  font-family: monospace;
+  content: "Error: Inconsistent indentation: 1 space was used for indentation, but the rest of the document was indented using 4 spaces.\A         on line 9 of /app/website/website_sale_collapse_categories/static/src/css/website_sale.sass\A \A 4:             display: inline-block\A 5:             width: 80%\A 6:             padding-left: 3px\A 7: \A 8:         i.fa\A 9:  cursor: pointer"; }
```

Comment:
Oops, Sass auto-copiler complaining. I should not have committed that. Fixed now.